utils/_decorators.py

```diff
@@ -14,6 +16,14 @@ def configure(config: pytest.Config):
     pass  # nothing to do right now
 
 
+class DecoratorType(enum.Enum):
```

Comment:
Prevent this to be exported by error, and use `StrEnum` (less prone to typo error when values are `str`) : 

```suggestion
class _DecoratorType(enum.StrEnum):
```

We're getting close !